src/main/WorkflowSelection.tsx

```diff
@@ -71,7 +70,7 @@ const WorkflowSelection : React.FC<{}> = () => {
   };
 
   // Layout template
-  const render = (topTitle: string, topText: {} | null | undefined, hasWorkflowButtons: boolean,
+  const render = (topTitle: string, topText: JSX.Element | null | undefined, hasWorkflowButtons: boolean,
```

Comment:
Not important but I simply noticed: it's probably fine to remove the `null | undefined` as the function is just called at three places, each time passing a non-null value.

```suggestion
  const render = (topTitle: string, topText: JSX.Element, hasWorkflowButtons: boolean,
```